language/tools/move-mv-llvm-compiler/tests/rbpf-tests.rs

```diff
@@ -34,7 +34,7 @@ use test_common as tc;
 
 pub const TEST_DIR: &str = "tests/rbpf-tests";
 
-datatest_stable::harness!(run_test, TEST_DIR, r".*\.move$");
+datatest_stable::harness!(run_test, TEST_DIR, r"rbpf-tests/[a-z0-9-_]*\.move$");
```

Comment:
This pattern had to be changed because running `move-build` while passing it the stdlib dependencies causes it to emit a "generated_interface_files" directory that contains additional move files. These files are detected by the `harness!` macro as tests.